dair_pll/drake_utils.py

```diff
@@ -36,16 +41,27 @@
 from pydrake.symbolic import Expression  # type: ignore
 from pydrake.systems.analysis import Simulator  # type: ignore
 from pydrake.systems.framework import DiagramBuilder  # type: ignore
-from pydrake.systems.meshcat_visualizer import MeshcatVisualizer  # type: ignore
+from pydrake.visualization import VideoWriter  # type: ignore
 
 from dair_pll import state_space
+from dair_pll import file_utils
 
 WORLD_GROUND_PLANE_NAME = "world_ground_plane"
 DRAKE_MATERIAL_GROUP = 'material'
 DRAKE_FRICTION_PROPERTY = 'coulomb_friction'
 N_DRAKE_FLOATING_BODY_VELOCITIES = 6
 DEFAULT_DT = 1e-3
 
+GROUND_COLOR = np.array([0.5, 0.5, 0.5, 0.1])
+
+CAM_FOV = np.pi/5
+VIDEO_PIXELS = [480, 640]
+FPS = 30
+SENSOR_POSE = RigidTransform(RollPitchYaw([-np.pi/2, 0, np.pi/2]), [1, 0, 0.2])
+EXP_NAME = os.environ['PLL_EXPERIMENT']
```

Comment:
why is the storage name getting hard-corded here?

Reply:
I ultimately left this line but moved it to file_utils.py.  I see it looks out of place since `PLL_EXPERIMENT` as an environment variable is an artifact of other features in my fork and doesn't appear elsewhere in the main branch here.  I didn't see a cleaner way of putting this somewhere temporary since none of the drake_utils.py functions get anything like `storage_name` as an argument.  As written, this will result in a file path for `output.gif` as the following **when `PLL_EXPERIMENT` is set:**
```
/path/to/dair_pll/results/pll_exp_name/temp/output.gif
```
...or instead **when `PLL_EXPERIMENT` is unset:**
```
/path/to/dair_pll/results/temp/output.gif
```

If you have a preferred way of choosing a `VIDEO_FILENAME`, feel free to suggest them here.

Reply:
This solution plus a `TODO` comment nearby in `file_utils` to find a more elegant solution in the future seems sufficient for now to me.